exercises/content-curation.md

```diff
@@ -0,0 +1,28 @@
+### Exercise
+
+This exercise is to assess your detective skills, and your content curation skills. It is similar to a typical task you would work on at Snyk.
+
+You are given a CVE ID and asked to discover all information on it. You do not need prior knowledge of security exploits, but it may help.
```

Comment:
Prepend with context, i.e. `Vulnerabilities are curated by many organisations, CVE being the most popular. Every public vulnerability gets an ID in the CVE database.`